src/SDKs/DataFactoryV1Extended/Microsoft.Azure.Management.DataFactories.Extended/Microsoft.Azure.Management.DataFactories.Extended.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactories.Extended</PackageId>
```

Comment:
@nicolaszhang I am not sure I understand this completely.
If you change a public API, you are breaking everyone who is using it.
Why can't you release nuget with a major version bump, this way whoever wants to adopt to breaking change will upgrade, who don't want to, they can stay on older version?
I am not sure why change name of nuget, that part I am still unclear.
Do you have a scenario where you want your customer to use old Hydra based nuget SxS (side by side) with the breaking change nuget?